src/main/resources/lib/guillotine/generic/input-types.js

```diff
@@ -627,6 +627,72 @@ function createInputTypes(context) {
             },
         }
     });
+
+    const highlightCommonFields = createHighlightCommonFields(context);
+
+    context.types.highlightPropertiesInputType = createHighlightPropertiesInputType(context, highlightCommonFields);
+
+    context.types.highlightInputType = createHighlightInputType(context, highlightCommonFields);
+}
+
+function createHighlightCommonFields(context) {
+    return {
+        encoder: {
```

Comment:
I kept encoder and tagsSchema outside the common fields

https://developer.enonic.com/docs/xp/stable/storage/highlighting
Highlighting settings can be set on a global level and then overridden on the property level (except tagsSchema and encoder, these two can be set only on the global layer).

https://github.com/enonic/app-patchman/blob/952317ed2ce7c90d27c54545368e6b7c811d7951/src/main/resources/admin/tools/patchman/graphql/inputTypes/highlight/addInputTypeHighlight.ts#L36

Reply:
Good catch!